OpenRA.Mods.Common/Effects/Missile.cs

```diff
@@ -76,6 +76,9 @@ public class MissileInfo : IProjectileInfo
 		[Desc("Vertical rate of turn.")]
 		public readonly int VerticalRateOfTurn = 6;
 
+		[Desc("Gravity applied while in free fall.")]
```

Comment:
Okay, that is a fair point.
I'd rather we come up with a better name (something implying a force acting on the missile) and use the values given to us.

But if you're against that and others are, no problem, just please update the description.